ObstacleMSG[] obstacles
float64 x
float64 y
float64 oriantation
float64 v
float64[] goals_x
float64[] goals_y
float64[] static_x
float64[] static_y
float64[] static_n
int64 n_trajectories
TrajectoryMSG truth
---
bool success
TrajectoryMSG[] trajectories
int64 truth_idx